src/pages/homePage.jsx

```diff
@@ -0,0 +1,80 @@
+import React, {  } from "react";
+import { Avatar, List, Space, Typography, Calendar } from "antd";
+import { MessageOutlined, StarOutlined } from "@ant-design/icons";
+const { Text } = Typography;
+
+const Home = (props) => {
+  const listData = [];
+  for (let i = 0; i < 6; i++) {
+    listData.push({
+      title: `Event Name Text`,
+      avatar:
+        "https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png",
+      content:
+        "We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure), to help people create their product prototypes.",
+    });
+  }
+
+  const IconText = ({ icon, text }) => (
+    <Space>
+      {React.createElement(icon)}
+      <Text>{text}</Text>
+    </Space>
+  );
+
+  return (
+    <div className="row-container">
+      <div className="card">
+        <List
+          itemLayout="vertical"
+          size="large"
+          pagination={{
+            // onChange: (page) => {
+            //   console.log(page);
+            // },
+            pageSize: 3,
+          }}
+          dataSource={listData}
+          renderItem={(item, index) => (
+            <List.Item
+              key={index}
```

Comment:
Try not to use the index as the key in React. Reference: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318